src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -431,6 +431,11 @@ internal enum WellKnownMember
         System_Runtime_CompilerServices_TupleElementNamesAttribute__ctorTransformNames,
 
         System_String__Format_IFormatProvider,
+        System_String__op_Implicit_ToReadOnlySpanOfChar,
+
+        System_String__Concat_2ReadOnlySpans,
+        System_String__Concat_3ReadOnlySpans,
+        System_String__Concat_4ReadOnlySpans,
```

Comment:
Since these are `string` members, they are really supposed to be SpecialMembers. 

It looks like it is simply an oversight that ```System_String__Format_IFormatProvider``` is here. ```WellKnownType.System_IFormatProvider``` is used only for the purpose of describing ```System_String__Format_IFormatProvider```'s signature. So , there should be no problem with making it a ```SpecialType```. I am not suggesting we do that in this PR.

Simply making ```ReadOnlySpan``` special is probably not an option. we might still want to recognize custom implementations of the type. I need to think more about this problem, but I am pretty sure we will have to do something about the new members added here.

Reply:
What's the difference between special types/members and well-known ones? On surface they both seem like symbols compiler uses internally to emit code. I placed these in well-known members just because all span-related members (constructors, properties etc.) seem to be there

Reply:
> What's the difference between special types/members and well-known ones

Special types are types that must be defined in core library, compiler will not look for them anywhere else. Special members are members of special types. Well known types can be found elsewhere, even in the source code.